app/pages/settings/change-password-form.jsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import auth from 'panoptes-client/lib/auth';
+import counterpart from 'counterpart';
+
+const MIN_PASSWORD_LENGTH = 8;
+
+export default class ChangePasswordForm extends React.Component {
+
+  constructor(props) {
+    super(props);
```

Comment:
`props` only needs to be passed into the `super()` call if `this.props` is being used in the constructor, so this can just be `super()`